app/graphql/irida_schema.rb

```diff
@@ -62,17 +64,19 @@ def self.object_from_id(global_id, ctx = {})
     gid = parse_gid(global_id, ctx)
 
     GlobalID.find(gid)
+  rescue ActiveRecord::RecordNotFound
```

Comment:
**nitpick (blocking):** Can you move this outside into a `rescue_from` block instead. So add a `rescue_from(ActiveRecord::RecordNotFound)` and raise the `coercion error there.

Reply:
Changed in https://github.com/phac-nml/irida-next/pull/669/commits/e2ce9e21635e7e0211fc843de561d8dfe496e2a2